### ISSUE TITLE
Vorta automatically adding itself to Login Items in macOS

### ISSUE THREAD

**Issue by apsessa** [borgbase/vorta#1508]
### Description

Everytime I start Vorta, it adds itself to the Login Items, ignoring that the option "Automatically start Vorta at login" is not checked. 

I run Vorta using a script that detects whenever my external media hard drive is mounted and then it runs Vorta. Since my computer takes some time to mount both the encrypted external disk and the remote disk where the backups go, running Vorta on login will cause an error since both the files to back up and the destination folders are not yet mounted and present. 

Both running the app with the script and doing so manually without script will add Vorta to the Login Items. I found nothing useful in the logs.

### Reproduction

- [X] I tried to reproduce the issue.
- [X] I was able to reproduce the issue.

### OS

macOS Ventura 13.0

### Version of Vorta

0.8.9

### What did you install Vorta with?

Binary

### Version of Borg

1.2.0

**Comment by real-yfprojects** [comment]
Thank you for reporting this issue. 
I have no idea why this is the case. It might be connected to vorta using the deprecated api.

**Comment by m3nu** [comment]
Probably due to the API used. Seeing the same with other apps.

**Comment by real-yfprojects** [comment]
Probem is: I have yet to find the new api for this.

**Comment by m3nu** [comment]
No, it's because we always add a login item and maybe remove it after that. Was easier to do back then.

Here a different implementation that doesn't trigger the warning: https://github.com/borgbase/vorta/pull/1549
